src/container/container.ts

```diff
@@ -249,6 +249,11 @@ class Container implements interfaces.Container {
         return bound;
     }
 
+    // check binding dependency only in current container
+    public isCurrentBound(serviceIdentifier: interfaces.ServiceIdentifier<any>): boolean {
```

Comment:
```suggestion
    public isCurrentBound<T>(serviceIdentifier: interfaces.ServiceIdentifier<T>): boolean {
```

What about this? We try not to use any these days